### ISSUE TITLE
Doesn’t work on Travis CI (node 0.10.x), throws Unhandled rejection TypeError

### ISSUE THREAD

**Issue by jirutka** [jedmao/eclint#30]
```
$ eclint check $(git ls-files)
Unhandled rejection TypeError: Object # h has no method 'equals'
    at detectCharset (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/linez/js/linez.js:35:41)
    at linez (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/linez/js/linez.js:19:19)
    at /home/travis/.nvm/v0.10.36/lib/node_modules/eclint/js/eclint.js:69:27
    at tryCatcher (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/promise.js:503:31)
    at Promise._settlePromiseAt (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/promise.js:577:18)
    at Async._drainQueue (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/async.js:128:12)
    at Async._drainQueues (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/async.js:133:10)
    at Async.drainQueues (/home/travis/.nvm/v0.10.36/lib/node_modules/eclint/node_modules/editorconfig/node_modules/bluebird/js/main/async.js:15:14)
    at process._tickCallback (node.js:442:13)
```

**Comment by jedmao** [comment]
This issue should be filed with the linez project.

**Comment by jedmao** [comment]
BTW, this is using [Buffer#equals](https://nodejs.org/api/buffer.html#buffer_buf_equals_otherbuffer) and all the tests pass fine on Travis for the linez project. If you can submit a breaking build I'll be happy to fix it.

**Comment by jirutka** [comment]
It’s [this build](https://travis-ci.org/asciidoctor/sublimetext-asciidoc/builds/73414200) from [this branch](https://github.com/asciidoctor/sublimetext-asciidoc/tree/eclint), eclint is called in [script/test:17](https://github.com/asciidoctor/sublimetext-asciidoc/blob/8d028a0a541bdd285bf5d21bd756188461c9315e/script/test#L17).

**Comment by jedmao** [comment]
Again, this is an issue with the [linez](https://github.com/jedmao/linez) project. Please file the issue there.

**Comment by jirutka** [comment]
So what does your previous comment mean? o.O

> If you can submit a breaking build I'll be happy to fix it.

**Comment by jedmao** [comment]
I'm the owner of both projects. Perhaps I should have said a breaking test.

**Comment by jedmao** [comment]
Real fix in v1.1.0: https://github.com/jedmao/eclint/commit/f3a87fb382d53ae4f2f8eb694556bda5d208d850
